crates/controller/src/command.rs

```diff
@@ -146,6 +163,34 @@ fn right_click_handler(
     }
 }
 
+fn double_click_handler(
+    keys: Res<Input<KeyCode>>,
+    pointer: Res<Pointer>,
+    playable: Query<&ObjectType, With<Playable>>,
+    drafts: Query<(), With<Draft>>,
+    mut select_in_rect_events: EventWriter<SelectInRectEvent>,
+) {
+    if !drafts.is_empty() {
+        return;
+    }
+    let selection_mode = if keys.pressed(KeyCode::LControl) {
+        SelectionMode::AddToggle
```

Comment:
I would use `Add` instead of `AddToggle` (the latter makes sense for single click but not double click).